chain/zcash/utxo.go

```diff
@@ -78,7 +78,7 @@ func (txBuilder TxBuilder) BuildTx(inputs []utxo.Input, recipients []utxo.Recipi
 
 	// Outputs
 	for _, recipient := range recipients {
-		addr, err := DecodeAddress(string(recipient.To))
+		addr, err := decodeAddress(string(recipient.To))
```

Comment:
This is why `AddressDecoder.DecodeAddress` should just call `decodeAddress`. Otherwise, we must remember to change _both_ implementations whenever we make changes. If we forget, or make a mistake while doing it, then addresses will end up being decoded differently in different places.